translate/pb/pb_decoder_entry.go

```diff
@@ -74,3 +57,14 @@ var StdProtobufDecoder = rel.NewNativeFunction("decode", func(param rel.Value) (
 		}), nil
 	}), nil
 })
+
+var StdProtobufDescriptor = rel.NewNativeFunction("decode", func(param rel.Value) (rel.Value, error) {
+	definitionBytes, isBytes := param.(rel.Bytes)
+	if !isBytes {
+		return nil, fmt.Errorf("//encoding.proto.descriptor: param not bytes")
+	}
+
+	return rel.NewTuple(rel.NewAttr(fileDescriptorSet, definitionBytes)), nil
+})
+
+const fileDescriptorSet = "FileDescriptorSet"
```

Comment:
Nit: put this up the top of the file. Otherwise a reader doesn't know it exists until after it's been used.